docs/source/quickstart.rst

```diff
@@ -1,81 +1,187 @@
+
+By entering the ``figs`` folder you'll find a range of diagnostic figures. For example, on the left hand side of the figure copied below, the background subtracted 2D spectrum for the first integration is plotted (top) alongside a 2D image of the estimated background. Note that the distinct striping is a result of 1/f noise in the NIRSpec detector electronics, and is dominant along pixel columns as they correspond to the direction of the detector readout. 
+
+To the right you can see a 2D representation of the variation in flux between consecutive integrations as a function of wavelength. In fact, the transit of WASP-39b can be seen via the horizontal band of reduced flux between integrations ~9-25. At the top, the median absolute deviation (MAD) for the entire dataset is displayed, and is calculated by determining the flux difference between each image and the next, for each wavelength, followed by taking the overall median of these values across all wavelengths and all images. 
+
+Finally, note that the actual data for these produced 1D spectra are contained in the ``*Table_Save.txt`` file.
+
+.. image:: ../media/stage3_quickstart.png
+
+Stage 4: Create Lightcurves
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Stage 4 takes the 1D spectra produced by the previous stage and turns them in to light curves. The number of wavelength channels to turn into light curves, along with the wavelength range across which they will be calculated, can be defined in the Stage 4 ``.ecf`` file. In the interest of reducing the computational burden of the following light curve fitting stage, only two light curves will be generated corresponding to 1.5-3.0 μm and 3.0-4.5 μm (see figure below). 
+
+Similarly to Stage 3, the actual data for the produced light curves can be found in the ``*Table_Save.txt`` file.
+
+.. image:: ../media/stage4_quickstart.png
+
+Stage 5: Lightcurve Fitting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Stage 5 takes all of the lightcurves produced by the previous stage and performs a variety of fitting routines to estimate specific system and planetary properties. For this quickstart, the fitting was performed using nested sampling
+as implemented by ``dynesty``, for a model assuming a transit of WASP-39b plus an aribitrary linear polynomial trend. 
+
+As a reminder, the input initial guesses and priors for the model properties are contained within the Stage 5 ``.epf`` file. To facilitate this quickstart demo, input parameters applicable to WASP-39b have already been assigned. For your own reductions, you'll need to tailor this file to the system you are observing and the type of fit you want to perform. 
+
+Additionally, nested sampling is not the only method of sampling the posterior distributions, and MCMC as implemented by ``emcee`` can also be used. Gaussian processes as implemented by ``george`` can also be included in the overall fitting routine. 
```

Comment:
I would remove the comment about george from this paragraph (can keep in a separate paragraph talking about different models one can use if you want), and instead note the presence of the lsq fitter in this paragraph. I would recommend here that people first try things with lsq and then only run dynesty when they have something they think will work well - otherwise they may wait an hour just to find out that the plots are all horrible, make a single tweak, and then repeat